public/app/plugins/panel/graph3/plugins/AnnotationsPlugin.tsx

```diff
@@ -29,8 +29,10 @@ export const AnnotationsPlugin: React.FC<AnnotationsPluginProps> = ({ annotation
     [timeZone]
   );
 
-  const annotationEventsStream = useMemo(() => getAnnotationsFromData(annotations), [annotations]);
-  const annotationsData = useObservable<AnnotationEvent[]>(annotationEventsStream);
+  const annotationsData = useMemo(() => {
+    return getAnnotationsFromData(annotations);
+  }, [annotations]);
+
```

Comment:
This seems like an odd change. Merge gone bad?

Reply:
@papagian?

Reply:
@hugohaggmark @papagian Now I'm wondering if it's something that happened during a merge of mine. I guess changes to this file can just be reverted?

Reply:
I would say yes @aknuds1

Reply:
I pushed a reversal of the file since I think it was probably changed during a merge of my own.

Reply:
Looks good @aknuds1 👍